src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -290,16 +290,7 @@ export function checkEligibility({ location, showModal }) {
         showModal,
       });
 
-      try {
```

Comment:
This block is not necessary, because we already have facility details for each facility on this page. This wasn't true don the old facility page, but that's now been deleted.